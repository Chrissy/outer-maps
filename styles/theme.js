import spacingScale from "./spacingScale";
import typeScale from "./typeScale";

export default {
  brandColor: "#344632",
  accentColor: "#FF6000",
  accentColorTintLighter: "#FFAE86",
  accentColorTintLight: "#FF8244",
  accentColorTintDark: "#CA4300",
  accentColorTintDarker: "#9B3300",
  gray1: "#F7F7F7",
  gray2: "#F3F3F3",
  gray3: "#E9E9E9",
  gray4: "#D8D8D8",
  gray5: "#A7A7A7",
  gray6: "#838383",
  gray7: "#494949",
  gray8: "#292929",
  blue: "#3232FF",
  purple: "#5700EC",
  orange: "#FF9500",
  red: "#D0011B",
  ss: spacingScale,
  ts: typeScale,
  headlineFont: "\"Hoefler Text A\", \"Hoefler Text B\"",
  bodyFont:
    "\"Gotham Narrow A\", \"Gotham Narrow B\", -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Helvetica, Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\""
};
