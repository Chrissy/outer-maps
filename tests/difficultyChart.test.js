import React from "react";
import theme from "../styles/theme";
import { ThemeProvider } from "emotion-theming";
import DifficultyChart from "../components/difficultyChart.js";
import renderer from "react-test-renderer";

it("renders", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
        <DifficultyChart score={80} />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
