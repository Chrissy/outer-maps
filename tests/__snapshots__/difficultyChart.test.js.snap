// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="body flex strenuous"
>
  <div
    className="circle1"
  />
  <div
    className="circle2"
    style={
      Object {
        "transform": "rotate(333deg)",
      }
    }
  />
  <div
    className="innerCircle flex"
  >
    <div
      className="data"
    >
      <div
        className="label label"
      >
        Difficulty
      </div>
      <div
        className="score"
      >
        80
      </div>
      <div
        className="difficulty"
      >
        Strenuous
      </div>
    </div>
  </div>
</div>
`;
